scripts/vpn_scan.sh

```diff
@@ -1,20 +1,25 @@
 #!/bin/sh
 
 # use with:
-# $ cat scripts/vpn_scan.sh | docker exec -i test_network_1_tailscale_1 ash
+# $ cat scripts/vpn_scan.sh | docker exec -i test_network_1-tailscale-1 ash
 
 apk add netcat-openbsd
 apk add lsof
 # lsof -i -P -n | grep LISTEN | grep -Eo ':(.+?)\s' | cut -c 2-
 
 # scan local ports
-tailscale status | grep "$(hostname)" | awk '{print $1}' | while read line; do
-    echo "Scanning $line"
-    nc -z -v "$line" 1-65535 2>&1 | grep succeeded
-done
+# uncomment to test against localhost
```

Comment:
This part just scans the localhost of the current container due to the IP having a local route which means that it bypasses the firewall rules anyway so its not really helpful.